packages/k2-alpine/src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect } from 'react'
+import { ImageSourcePropType, ViewStyle } from 'react-native'
+import Animated, {
+  Easing,
+  useAnimatedStyle,
+  useSharedValue,
+  withTiming
+} from 'react-native-reanimated'
+import { BlurView } from 'expo-blur'
+import { View } from '../Primitives'
+import { useTheme } from '../..'
+import { HexagonImageView, HexagonBorder } from './HexagonImageView'
+
+export const Avatar = ({
+  source,
+  size,
+  isSelected,
+  isPressed,
+  hasBlur,
+  style
+}: {
+  source: ImageSourcePropType
+  size: number | 'small' | 'large'
+  isSelected?: boolean
+  isPressed?: boolean
+  hasBlur?: boolean
+  style?: ViewStyle
+}): JSX.Element => {
+  const { theme } = useTheme()
+  const blurAreaInset = 50
+
+  const height = typeof size === 'number' ? size : size === 'small' ? 90 : 150
+
+  const pressedAnimation = useSharedValue(1)
+  const pressedAnimatedStyle = useAnimatedStyle(() => ({
+    transform: [{ scale: pressedAnimation.value }]
+  }))
+  const surfacePrimaryBlurBg = theme.isDark ? '#050506' : undefined // to cancel out the blur effect on the $surfacePrimary, we need to use a darker background color for the blur view, only for dark mode
```

Comment:
I had to set a slightly darker background(`#050506`) beneath the blur view to eliminate the color difference between the blurred `$surfacePrimary` and the `$surfacePrimary`. This issue occurs only in dark mode. This screenshot illustrates the result when the darker background is not set(though the difference is very subtle).

![Simulator Screenshot - iPhone 15 Pro - 2024-11-29 at 14 42 18](https://github.com/user-attachments/assets/470337d4-a5ba-4fcc-868f-b51127c32de0)

Reply:
nice work around. can you let design know about this in case they want to change something?

Reply:
synced with design team.